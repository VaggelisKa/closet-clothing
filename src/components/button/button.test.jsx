import React from 'react';
import { shallow } from 'enzyme';
import Button from './button.component';

it('Should render button component', () => {
  expect(shallow(<Button />)).toMatchSnapshot();
});
