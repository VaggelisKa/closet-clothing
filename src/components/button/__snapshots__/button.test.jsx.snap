// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render button component 1`] = `<styled.button />`;
